DatadogRUM/Sources/Integrations/WebViewEventReceiver.swift

```diff
@@ -68,6 +73,14 @@ internal final class WebViewEventReceiver: FeatureMessageReceiver {
                     let serverTimeOffsetInMs = self.getOffsetInMs(viewID: viewID, context: context)
                     let correctedDate = Int64(date) + serverTimeOffsetInMs
                     event["date"] = correctedDate
```

Comment:
question/ Could `ViewCache` be used to cleanup this class from NTP offsets caching? I mean [this code](https://github.com/DataDog/dd-sdk-ios/blob/54fae19f7b45787301ae8d0b99d701fc0f69e9ba/DatadogRUM/Sources/Integrations/WebViewEventReceiver.swift#L97-L122), that has similar concepts to what we introduced in `ViewCache`: _get by ID_, _purge_ etc.

Reply:
The NTP caching is a simple `key->value` cache while the `ViewCache` will look for the closest view in the past using timestamp. So it's different key type and query that are not compatible :/